docs/release_notes.rst

```diff
@@ -13,6 +13,11 @@ EIA 860
 ~~~~~~~
 * Added EIA 860 final release data from 2023. See :issue:`3684` and PR :pr:`3871`.
 
+EIA 923
+~~~~~~~
+* Added EIA 923 final release data from 2023 and revised data from 2022. See
+  :issue:`3902` and PR :pr:`3903`.
+
```

Comment:
this is so silly, but I see that there are different tenses used throughout (above is says "Added" and below it says "Integrate") is there one that is used most commonly? Perhaps we can use that tense and fix the others.